src/app/App.js

```diff
@@ -141,6 +141,7 @@ export default class App {
   }
 
   _callInfo(initMethod, initArgs) {
-    return `${initMethod.name}(${initMethod.inputs.map(i => i.type).join(',')}) with ${JSON.stringify(initArgs).replace(/^\[|\]$/g, "")}`;
+    return `${initMethod.name} with
+${initMethod.inputs.map((input, index) => ` - ${input.name} (${input.type}): ${JSON.stringify(initArgs[index])}`).join('\n')}`;
```

Comment:
I think it will look better splitting this line as follows:
```js
const args = initMethod.inputs.map((input, index) => ` - ${input.name} (${input.type}): ${JSON.stringify(initArgs[index])}`)
return `${initMethod.name} with: \n${args.join('\n')}`
```